internal/worker_runner.go

```diff
@@ -299,3 +321,34 @@ func (wr *workerRunner) isOkToQueryOBD() (bool, api.PowerStatusResponse) {
 	}
 	return false, status
 }
+
+type SignalsQueue struct {
+	signals      []models.SignalData
```

Comment:
This would be our queue, which would be populated with signal pids values in parallel